pkgs/by-name/dp/dprint/plugins/default.nix

```diff
@@ -26,6 +30,13 @@ let
         updateScript = ./update-plugins.py;
         inherit initConfig updateUrl;
       };
+      nativeBuildInputs = [ dprint ];
+      postFetch = ''
```

Comment:
If the hash or URL is changed in a plugin, this will work. In other cases, the cache will work.
Should I prefer `testers.invalidateFetcherByDrvHash` for this purpose rather than `postFetch`?
I wish to ensure that the checker will run when running nix-build, such as `versionCheckHook`.

Reply:
`nixpkgs-review` also returns no rebuild state for this PR.

Reply:
g-plane/* plugins all fail with formatting empty files

Reply:
Thanks for your check! I've just tested with biome and some official plugins. So added small files for each plugin to fix the pattern.